flixel/system/debug/watch/Watch.hx

```diff
@@ -136,7 +136,6 @@ class Watch extends Window
 		resetEntries();
 		minSize.setTo(getMaxMinWidth() + entriesContainerOffset.x, entriesContainer.height + entriesContainerOffset.y);
 		super.updateSize();
-		reposition(x, y);
```

Comment:
I didn't see any difference in behavior coming from this line, is there something i'm missing?

Reply:
the reposition() comes from the parent `Window.hx` class, which updates it's internal x/y position, as well as repositions the location to be within the window bounds. It's much more noticable if you have a lot of stuff in `FlxG.watch.add()` or whatever that'd overflow the screen. Calling `reposition()` will properly keep the watch window in-bounds, rather than overflowing off the screen where you need to drag it up to see all of the stuff